rego/rego.go

```diff
@@ -601,6 +602,20 @@ func PrintTrace(w io.Writer, r *Rego) {
 	topdown.PrettyTrace(w, *r.tracebuf)
 }
 
+// UnsafeBuiltins is a helper that sets a set of unsafe builtins on the
+// compiler.
```

Comment:
Nit: it might be worth just adding another sentence like

```
If a rule or query is found to refer to an unsafe a built-in function, an error will be returned when the Rego object is executed.
```